llvm/test/tools/sycl-post-link/assert-indirect-with-split-2.ll

```diff
@@ -85,6 +78,13 @@ entry:
   ret void
 }
 
+; CHECK-NOT: main_TU0_kernel1
+define dso_local spir_kernel void @main_TU0_kernel1() #0 {
+entry:
+  call spir_func void @_Z4foo1v()
+  ret void
+}
+
```

Comment:
Why this change is needed?

Reply:
New approach lead to a change in the order of kernel names. Because of that I replaced CHECK: with CHECK-DAG:. And it is not allowed to mix CHECK-DAG with CHECK-NOT. That's why I moved some function from DAG sections.